pages/reference/OS/advanced.md

```diff
@@ -22,7 +22,7 @@ $ cat /mnt/boot/config.txt
 
 ### Modifying `config.txt` locally before the first boot
 
-Before the device is [provisioned][device-provisioning], you may edit `config.txt` by mounting a flashed SD card (with the partition label `resin-boot`) and editing the file directly. This will only work if you edit the file before the device's first boot, as after device provisioning any changes will be overwritten by the device supervisor with values read from the {{ $names.cloud.lower }} API.
+Before the device is [provisioned][device-provisioning], you may edit `config.txt` by mounting a flashed SD card (with the partition label `resin-boot`) and editing the file directly. Any values added to `config.txt` will be added to the dashboard during device provisioning. This will only work if you edit the file before the device's first boot, as after device provisioning, any changes will be overwritten by the device supervisor with values read from the {{ $names.cloud.lower }} API.
```

Comment:
In one sentence it refers to "the dashboard" and in another sentence it refers to "the balena API". I guess these are more or less the same thing, but perhaps this could be clarified / tidied up?